src/Components/Style.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button } from "antd";
+import styled from "@emotion/styled";
+import { Menu } from "antd";
+
+export const FormattedMenu = styled(Menu)(() => ({
+  display: "flex",
+  justifyContent: "right",
+}));
+
+export const UserDiv = styled("div")(() => ({
+  display: "flex",
+  justifyContent: "right",
+  marginRight: "15px",
+  padding: "10px",
+  gap: "10px",
```

Comment:
Nem neztem bele a felhasznalas helyen reszletesen, mert nem cloneoztam le a report es nem tudom MEG(!) fejben renderelni a UI-t, viszont itt ha jol gondolom az tortenik, hogy az elemek kozott van 10 pixel gap, aztan jobb oldalon meg tovabbi 15px, illetve az elemeknek van egy belso paddingje is, ami 10px. Majdnem teljesen biztos vagyok benne, hogy a tervezett kinezetet el lehet erni a 3 property helyett maximum 2-vel, akar 1-el, de ez kicsit mar a szorszalhasogatas kategoriaja.